gradio/external.py

```diff
@@ -49,7 +49,7 @@ def encode_to_base64(r: requests.Response) -> str:
     pipelines = {
         "audio-classification": {
             # example model: https://hf.co/ehcalabres/wav2vec2-lg-xlsr-en-speech-emotion-recognition
-            "inputs": components.Audio(label="Input", source="upload", type="filepath"),
+            "inputs": components.Audio(source="upload", type="filepath", label="Input"),
```

Comment:
Out of curiosity, why do we need to reorder these?

Reply:
It probably happened automatically with refactoring.